tests/test_get.py

```diff
@@ -45,21 +47,23 @@ def test_get_calls_correct_url(mocker):
     except ValueError:
         pass
 
-    mock_req.get.assert_called_with("www.foo.com?bar=baz")
+    mock_req.get.assert_called_with("http://www.foo.com?bar=baz")
 
 
-def test_get_raises_errors(mocker):
+@responses.activate
+def test_get_server_error(mocker):
     # mock the ENDPOINTS dictionary
     mocker.patch.dict("statsapi.ENDPOINTS", fake_dict(), clear=True)
-    # mock the requests object
-    mock_req = mocker.patch("statsapi.requests", autospec=True)
-    # mock the status code to always be 200
-    mock_req.get.return_value.status_code = 0
+    responses.add(responses.GET, 'http://www.foo.com?bar=baz', status=500)
 
-    # bad status code
-    with pytest.raises(ValueError):
+    with pytest.raises(requests.exceptions.HTTPError):
         statsapi.get("foo", {"bar": "baz"})
 
+def test_get_invalid_endpoint(mocker):
+    # mock the ENDPOINTS dictionary
+    mocker.patch.dict("statsapi.ENDPOINTS", fake_dict(), clear=True)
+    # mock the requests object
+    mock_req = mocker.patch("statsapi.requests", autospec=True)
     # invalid endpoint
     with pytest.raises(ValueError):
```

Comment:
Shouldn't this also be updated to `requests.exceptions.HTTPError` as well? I honestly still don't fully understand how this testing works, but I think this is another call to `statsapi.get()` which will now return an HTTPError.

```suggestion
    with pytest.raises(requests.exceptions.HTTPError):
```

Reply:
I think this test case is meant to cover the case where a totally unsupported endpoint is requested via `get`. [That code path](https://github.com/toddrob99/MLB-StatsAPI/blob/f4f7ebdb64049b0b02bcedb662d98316cae544bd/statsapi/__init__.py#L1530) still raises a `ValueError` and it happens before the HTTP request is made, so I think this test shouldn't be changed.

Reply:
You're right, that's a `ValueError` thrown by the wrapper and not an error status from the API. Good catch.